api/src/wfl/module/xx.clj

```diff
@@ -74,15 +74,16 @@
 
 ;; visible for testing
 (defn make-inputs-to-save [output-url inputs]
-  (let [sample-name (fn [basename] (first (str/split basename #"\.")))
-        [_ path] (gcs/parse-gs-url (some inputs [:input_bam :input_cram]))
-        basename    (or (:base_file_name inputs) (util/basename path))]
+  (let [sample (some inputs [:input_bam :input_cram])
+        [_ base _] (all/bam-or-cram? sample)
+        leaf   (util/basename base)
```

Comment:
Looks like the docstring for util/basename is wrong, or it is misnamed.
Maybe it should be named util/leafname or otherwise fixed.

https://github.com/broadinstitute/wfl/blob/0a7f80ea0255f576dc45d0994af1da823df69c37/api/src/wfl/util.clj#L102